collections/src/main/scala/strawman/collection/SeqView.scala

```diff
@@ -0,0 +1,41 @@
+package strawman
+package collection
+
+import scala.{Int, IndexOutOfBoundsException}
+
+trait SeqView[+A] extends SeqOps[A, View, View[A]] with View[A] {
+
+  override def view: SeqView[A] = this
+
+  override def map[B](f: A => B): SeqView[B] = new SeqView.Map(this, f)
+
+  override def prepended[B >: A](elem: B): SeqView[B] = new SeqView.Prepended(elem, this)
+
+  override def take(n: Int): SeqView[A] = new SeqView.Take(this, n)
+
```

Comment:
`patch` is used in a some partest test https://github.com/scala/scala/blob/2.13.x/test/files/run/t3511.scala#L32